lib/kb_Metrics/metricsDBs.py

```diff
@@ -1,4 +1,4 @@
-
+import datetime
```

Comment:
is it used anywhere? if not just remove.

Reply:
Ha...I was using it for rewriting the query and then abandoned that effort without removing this import.  I will remove it.